litmus-portal/frontend/src/pages/ApplicationDashboard/index.tsx

```diff
@@ -40,10 +40,10 @@ import * as DashboardActions from '../../redux/actions/dashboards';
 import * as DataSourceActions from '../../redux/actions/dataSource';
 import { RootState } from '../../redux/reducers';
 import { getProjectID } from '../../utils/getSearchParams';
-import DataSourceInactiveModal from '../../views/AnalyticsDashboard/MonitoringDashboardPage/DataSourceInactiveModal';
-import InfoDropdown from '../../views/AnalyticsDashboard/MonitoringDashboardPage/InfoDropdown';
-import DashboardPanelGroup from '../../views/AnalyticsDashboard/MonitoringDashboardPage/Panel/DashboardPanelGroup';
-import TopNavButtons from '../../views/AnalyticsDashboard/MonitoringDashboardPage/TopNavButtons';
+import DataSourceInactiveModal from '../../views/Analytics/ApplicationDashboard/DataSourceInactiveModal';
```

Comment:
Can you please remove the eslint disable from this file?

Reply:
This page will be refactored it will be updated after this beta.